resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -357,110 +333,73 @@ tlmgr install eurosym \
-```
-
-Then, 
-```
-wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
-sudo apt-get update
-sudo apt-get install postgresql-12
+  upquote
 ```
-That's it! You can test if the installation worked by running
-```
-sudo su -c psql postgres
-```
-
-From a succesful install, the above should yield the prompt to change to what is shown below:
-```
-psql (12.3 (Ubuntu 12.3-1.pgdg18.04+1))
-Type "help" for help.
-postgres=# 
-
-```
-
-> Note: to exit from Postgres type `exit`
-
-
 
-## LaTeX
-Open Terminal and type the following command:
-```
-sudo apt-get install texlive
-```
+To test that your latex installation is working with jupyter notebooks,
+launch `jupyter lab` from a terminal and open either a new notebook
+or the same one you used to test IRkernel above.
+Go to `File -> Export notebook as... -> Export Notebook to PDF`.
+If the PDF file is created,
+your LaTeX environment is setup correctly.
+You can also check by typing the following to ask for the version of latex:
 
-After installation, in terminal type the following to ask for the version:
 ```
 latex --version
 ```
-you should see something like this if you were successful:
+
+You should see something like this if you were successful:
 
 ```
-pdfTeX 3.14159265-2.6-1.40.18 (TeX Live 2017/Debian)
-kpathsea version 6.2.3
-Copyright 2017 Han The Thanh (pdfTeX) et al.
+pdfTeX 3.14159265-2.6-1.40.21 (TeX Live 2020)
+kpathsea version 6.3.2
+Copyright 2020 Han The Thanh (pdfTeX) et al.
 There is NO warranty.  Redistribution of this software is
 covered by the terms of both the pdfTeX copyright and
 the Lesser GNU General Public License.
 For more information about these matters, see the file
 named COPYING and the pdfTeX source.
 Primary author of pdfTeX: Han The Thanh (pdfTeX) et al.
-Compiled with libpng 1.6.34; using libpng 1.6.34
-Compiled with zlib 1.2.8; using zlib 1.2.8
-Compiled with poppler version 0.62.0
+Compiled with libpng 1.6.37; using libpng 1.6.37
+Compiled with zlib 1.2.11; using zlib 1.2.11
+Compiled with xpdf version 4.02
 ```
 
-## Docker
-
-You will use Docker to create reproducible, sharable and shippable computing environments for your analyses. For this you will need a Docker account. You can sign up for a free one [here](https://store.docker.com/signup?next=%2F%3Fref%3Dlogin).
+## PostgreSQL
 
-After signing-up, you also need to install Docker **CE** for Ubuntu. Follow the installation instructions using the "Install using the repository" methods found [here](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository).
+We will be using PostgreSQL as our database management system. To install PostgreSQL type the following command:
 
-Next, follow the Linux post installation steps [here](https://docs.docker.com/engine/install/linux-postinstall/) so that you can run Docker without typing `sudo`. 
+```
+sudo apt install postgresql postgresql-contrib
```

Comment:
postgres 12 is now in the repos so I don't think we need to add the separate key and repo unless there was another reason for that then to get a recent version

Reply:
Nope, just my lack of ubuntu knowledge! Please go ahead and fix this.